examples/opentelemetry/main.go

```
@@ -0,0 +1,292 @@
+		log.Println(err)
+	}
+	if err := itbl.AddFieldColumn("host", types.STRING); err != nil {
+		log.Println(err)
+	}
+	if err := itbl.AddFieldColumn("temperature", types.FLOAT); err != nil {
+		log.Println(err)
+	}
+	if err := itbl.AddTimestampColumn("timestamp", types.TIMESTAMP_MICROSECOND); err != nil {
+		log.Println(err)
+	}
+
+	if err := itbl.AddRow(1, "hello", 1.1, time1); err != nil {
+		log.Println(err)
+	}
+	if err := itbl.AddRow(2, "hello", 2.2, time2); err != nil {
+		log.Println(err)
+	}
+	if err := itbl.AddRow(3, "hello", 3.3, time3); err != nil {
+		log.Println(err)
+	}
+
+	utbl, err := table.New("monitors_with_schema")
+	if err != nil {
+		log.Println(err)
+	}
+
+	// add column at first. This is to define the schema of the table.
+	if err := utbl.AddTagColumn("id", types.INT64); err != nil {
+		log.Println(err)
+	}
+	if err := utbl.AddFieldColumn("host", types.STRING); err != nil {
+		log.Println(err)
+	}
+	if err := utbl.AddFieldColumn("temperature", types.FLOAT); err != nil {
+		log.Println(err)
+	}
+	if err := utbl.AddTimestampColumn("timestamp", types.TIMESTAMP_MICROSECOND); err != nil {
+		log.Println(err)
+	}
+
+	if err := utbl.AddRow(1, "hello", 1.2, time1); err != nil {
+		log.Println(err)
+	}
+
+	dtbl, err := table.New("monitors_with_schema")
+	if err != nil {
+		log.Println(err)
+	}
+
+	// add column at first. This is to define the schema of the table.
+	if err := dtbl.AddTagColumn("id", types.INT64); err != nil {
+		log.Println(err)
+	}
+	if err := dtbl.AddFieldColumn("host", types.STRING); err != nil {
+		log.Println(err)
+	}
+	if err := dtbl.AddFieldColumn("temperature", types.FLOAT); err != nil {
+		log.Println(err)
+	}
+	if err := dtbl.AddTimestampColumn("timestamp", types.TIMESTAMP_MICROSECOND); err != nil {
+		log.Println(err)
+	}
+
+	if err := dtbl.AddRow(3, "hello", 3.3, time3); err != nil {
+		log.Println(err)
+	}
+
+	return []*table.Table{itbl, utbl, dtbl}
+}
+
+func (c client) write(data *table.Table) {
```

Ditto. You can propagate the error to the caller.